doc/source/whatsnew/v0.18.0.txt

```diff
@@ -108,6 +108,9 @@ Other enhancements
 - A simple version of ``Panel.round()`` is now implemented (:issue:`11763`)
 - For Python 3.x, ``round(DataFrame)``, ``round(Series)``, ``round(Panel)`` will work (:issue:`11763`)
 - ``Dataframe`` has gained a ``_repr_latex_`` method in order to allow for automatic conversion to latex in a ipython/jupyter notebook using nbconvert. Options ``display.latex.escape`` and ``display.latex.longtable`` have been added to the configuration and are used automatically by the ``to_latex`` method.(:issue:`11778`)
+- sys.getsizeof(obj) returns the memory usage of a pandas object including the (non-object)
```

Comment:
OK, if it's a material performance drag and a different enough set of use cases, sobeit